pages/tools/airtable.md

```diff
@@ -6,14 +6,31 @@ redirect_from:
+[Airtable](https://airtable.com/) is a cloud-based tool that allows users to organize their workflow, data, and records in a flexible and visually appealing way without the need for any coding. **Licenses are now available for all TTS employees**. 
 
-Licenses that allow edit access are allocated on a per team basis. The license managers for each group are listed [here](https://docs.google.com/spreadsheets/d/1lbf8mkD4eJuKlRg5Uw2r1FHx3YCAiZp7P9qcoEG6rnk/edit#gid=0). If you are part of one of these teams, you can reach out to the listed point of contact in order to request one of the limited licenses.
+### How to Request a License 
 
-If you are on another team that does not yet have an allocation of Airtable licenses but is interested in using it, we will be rolling out a new process for requesting additional licenses in the summer of 2024.
+TTS employees can obtain a free, read-only license by simply making a request via the [#airtable-smartsheet](https://app.slack.com/client/E011QM4UN3G/C6YNTUPS9?space=1) Slack channel. Tech Operations will fulfill the request by granting the employee a read-only license. Read-only users can view but not edit or comment on everything in the workspace, base, or interface. 
 
-View-only licenses are free! Any existing Airtable user should be able to grant you one by default by sharing view-only access to an existing base, but should you run into any difficulty, you can reach out in [#airtable-smartsheet](https://app.slack.com/client/T025AQGAN/C6YNTUPS9). All users must log into Airtable using the Single Sign On option.
+Administrators, owners, creators, and editors all require paid licenses. TTS has a limited number of paid licenses available so please ensure you have a job-related need before requesting a paid license. You will need an email from your supervisor complete with the following information forwarded to tts-tech-operations@gsa.gov for your request to be approved. Your [Airtable group manager](https://docs.google.com/spreadsheets/d/1lbf8mkD4eJuKlRg5Uw2r1FHx3YCAiZp7P9qcoEG6rnk/edit?gid=0#gid=0) can help you if you have questions about any of these items:
 
-A more complete guide to Airtable, complete with training resources, can be found [here](https://airtable.com/appQbn3D4GMM8SD0O/pagjMkXUv7fCeTWIZ), though you will need to have at least a viewer licenses to Airtable in order to access it. More materials, including additional guidance on privacy, common use cases, and FAQs is coming soon.
+* The [workspace](https://airtable.com/appQbn3D4GMM8SD0O/pag7cdXd49Dat3bBw?detail=eyJwYWdlSWQiOiJwYWdxTEdwRFlFYktubXpMaiIsInJvd0lkIjoicmVjSVZzTU82QVkxUVhnbjEiLCJzaG93Q29tbWVudHMiOmZhbHNlLCJxdWVyeU9yaWdpbkhpbnQiOm51bGx9) you would like to be invited to initially
+* The level of access you need to the above workspace: owner, creator, editor, or viewer
+* A brief justification for your request (be mindful that each Airtable editor seat costs TTS nearly $800 per year)
```

Comment:
Suggest removing the $ amount here and just state that it cost TTS 

(be mindful that each Airtable editor seat costs TTS)